language-design.md

```diff
@@ -38,7 +69,14 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+A well-designed language is extendible, has a relatively small core vocabulary, is easy to learn (though not necessarily for everyone),
+is easy for the user to read and write, and has good documentation.
+On the other hand, a poorly designed language makes it difficult to write libraries and extentions,
+has inconsistent libraries, has a large and cumbersome core vocabulary, has a steep learning curve,
+or is otherwise difficult to use.
+These lists are general statements, of course, not universal; a well-designed language
+need not have all the elements in the first list, and a poorly designed language need not have
+any of the elements in the second list.
```

Comment:
You guys bring up extensibility - an idea closely related to the talk on language growth :smiley: